roles/openshift_on_openstack/tasks/main.yml

```diff
@@ -283,7 +283,7 @@
 # Run the Ansible playbook that installs OpenShift.
 - name: Installing OpenShift on OpenStack
   # Using two inventory flags here as the README instructs.
-  shell: "{{ ansible_playbook }} -vvv --user openshift -i inventory -i {{ inventory_py }} {{ openshift_openstack_dir }}/openshift-cluster/install.yml 2>&1 >> {{ openshift_install_log }}"
+  shell: pbench-user-benchmark --pbench-post='/usr/local/bin/scraper -i $benchmark_results_dir/tools-default -o $benchmark_results_dir -proc {{ process_names }}; ansible-playbook -i {{ pbcompare_inventory }} {{ pbcompare_playbook }} -e \'new_file='$benchmark_results_dir/out.json''' -- {{ ansible_playbook }} -vvv --user openshift -i inventory -i {{ inventory_py }} {{ openshift_openstack_dir }}/openshift-cluster/install.yml 2>&1 >> {{ openshift_install_log }}
```

Comment:
At which point are the pbench tools registered?  Don't see any pbench-register-tool anywhere?